app.py

```diff
@@ -0,0 +1,55 @@
+import connexion
+import os
+import json
+from jupyterhub_singleuser_profiles.profiles import SingleuserProfiles
+
+_PROFILES = SingleuserProfiles(verify_ssl=False)
+_PROFILES.load_profiles()
```

Comment:
I believe we said we need to run load_profiles before each request to make sure things are up to date - correct?

Maybe check this: https://github.com/zalando/connexion/issues/906